sql/mob_pools.sql

```diff
@@ -6050,12 +6050,12 @@ INSERT INTO `mob_pools` VALUES (5988,'makki-chebukki','Makki-Chebukki',153,0x000
 INSERT INTO `mob_pools` VALUES (5989,'king_of_hearts','KingOfHearts',61,0x0000230C00000000000000000000000000000000,5,3,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
 INSERT INTO `mob_pools` VALUES (5990,'morimar','Morimar',149,0x0000240C00000000000000000000000000000000,9,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
 INSERT INTO `mob_pools` VALUES (5991,'darrcuiln','Darrcuiln',489,0x0000250C00000000000000000000000000000000,1,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
-INSERT INTO `mob_pools` VALUES (5992,'aahm','ArkHM',149,0x0000290C00000000000000000000000000000000,13,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (5992,'aahm','ArkHM',149,0x0000290C00000000000000000000000000000000,1,13,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
```

Comment:
Don't forget the weapon type in the core for mobs is currently bs, and that field controls if a mob double swings or not. IIRC hume ark angel dual wields. So you'd have to set his type "wrong" here to get correct behavior in that case.

Reply:
"But /NIN gives Dual Wield!"
"Not until main level is 20."

If that's the case, then Luzaf and a few others might encounter difficulties too.

Reply:
@ibm2431 Mobs, pets, and trusts don't obey those rules so that won't come into play at all. Whatever the pool value says they should do, they will do at any level.

And on retail trusts that dual wield will do so at any level afaik (I haven't seen any example to the contrary).